README.md

```diff
@@ -69,7 +69,8 @@ Extensions can override the following default extension points to add or change
     onEvent : function(name, evt) {return true;},
```

Comment:
Btw, seems we also forgot the `init` function in there...
https://github.com/bigskysoftware/htmx/blob/master/src/htmx.js#L4791

The easiest thing to do here would probably to simply copy-paste the core's `extensionBase` literal object into this README

Reply:
Good Catch! I copied the `extensionBase` object over 😉